AppointmentReminders.Web/Content/bootstrap-datetimepicker.css

```diff
@@ -0,0 +1,372 @@
+/*!
```

Comment:
Versioning is much harder when you keep a local copy. Dependency changes
can break the application if not managed carefully. Can't we install the
packages with nuget or something? They have jQuery, Bootstrap and the
Bootstrap date picker. These are dependencies, not part of the application
and thus shouldn't be in the repository.

On Fri, Jul 17, 2015 at 11:16 AM, Agustin Camino <notifications@github.com>
wrote:

> In AppointmentReminders.Web/Content/bootstrap-datetimepicker.css
> <https://github.com/TwilioDevEd/appointment-reminders-csharp/pull/1#discussion_r34904873>
> :
>
> > @@ -0,0 +1,372 @@
> > +/*!
>
> I'll review the unused dependencies, and rid of them. To use a CDN or a
> local library is almost like a trivial decision, so I'll keep using a local
> copy, this shouldn't be a big deal, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TwilioDevEd/appointment-reminders-csharp/pull/1/files#r34904873>
> .
>